operate/webapp/src/main/java/io/camunda/operate/webapp/elasticsearch/backup/ElasticsearchBackupRepository.java

```diff
@@ -375,59 +440,41 @@ private GetBackupStateResponseDto getBackupResponse(Long backupId, List<Snapshot
   }
 
   /** CreateSnapshotListener */
-  private class CreateSnapshotListener implements ActionListener<CreateSnapshotResponse> {
+  public class CreateSnapshotListener implements ActionListener<CreateSnapshotResponse> {
 
     private final BackupService.SnapshotRequest snapshotRequest;
     private final long backupId;
     private final Runnable onSuccess;
     private final Runnable onFailure;
 
     public CreateSnapshotListener(
-        BackupService.SnapshotRequest snapshotRequest, Runnable onSuccess, Runnable onFailure) {
+        final BackupService.SnapshotRequest snapshotRequest,
+        final Runnable onSuccess,
+        final Runnable onFailure) {
       this.snapshotRequest = snapshotRequest;
-      this.backupId = Metadata.extractBackupIdFromSnapshotName(snapshotRequest.snapshotName());
+      backupId = Metadata.extractBackupIdFromSnapshotName(snapshotRequest.snapshotName());
       this.onSuccess = onSuccess;
       this.onFailure = onFailure;
     }
 
     @Override
-    public void onResponse(CreateSnapshotResponse response) {
+    public void onResponse(final CreateSnapshotResponse response) {
       handleSnapshotReceived(response.getSnapshotInfo());
     }
 
     @Override
-    public void onFailure(Exception ex) {
+    public void onFailure(final Exception ex) {
       if (ex instanceof SocketTimeoutException) {
         // This is thrown even if the backup is still running
         LOGGER.warn(
             String.format(
                 "Timeout while creating snapshot [%s] for backup id [%d]. Need to keep waiting with polling...",
```

Comment:
I would rename this `Timeout` text in the warning to `SocketTimeout`, to have a clear distinction between the SocketTimeout and the Polling Timeout.